pkg/processor/k8sprocessor/go.mod

```diff
@@ -104,3 +104,8 @@ require (
 	sigs.k8s.io/structured-merge-diff/v4 v4.2.3 // indirect
 	sigs.k8s.io/yaml v1.3.0 // indirect
 )
+
+// This component uses the otelcoltest package, which depends on zpagesextension, but uses a replace to refer to it in its go.mod
+// As a result, we pull the wrong version here without this replace
+// This can be removed once upstream updates the versions in zpagesextension go.mod or removes the dependency
+replace go.opentelemetry.io/collector/extension/zpagesextension => go.opentelemetry.io/collector/extension/zpagesextension v0.69.0
```

Comment:
Isn't this the same problem as with importing `sumologicextension` in `sumologicexporter`? I remember having similar issues when I used `sumologicexporter` externally. Idk if there is a reasonable solution for that, doesn't seem so.

Reply:
There probably isn't, but we have a new release fixing this here: https://github.com/open-telemetry/opentelemetry-collector/releases/tag/cmd%2Fbuilder%2Fv0.69.1, so I'll update this PR.

Reply:
Actually, it looks like 0.69.1 is not out for contrib, so we'd need to have two different versions for different components. I'll just leave this as is and add an issue to remove the replaces for 0.70.0.

Reply:
https://github.com/SumoLogic/sumologic-otel-collector/issues/894